### PR TITLE
Dcs 2239 risk management add pom consultation question

### PR DESCRIPTION
**Pull request by louisenorris**
Added question required to confirm COM has consulted POM to the new version of the risk management section of the licence:

**COM checks view page:**
Screenshot 2023-05-17 at 17 10 18

**Validation message also added as shown on the review page before a COM sends a case back to the CA:**
Screenshot 2023-05-17 at 17 12 01

**Review page for risk section also updated following updates made is a user selects `Change` on the COM review page:**
Screenshot 2023-05-17 at 17 12 25

**Task list now includes warning if user answers `No` to POM consultation question, also will display multiple warnings if required:**
Screenshot 2023-05-17 at 17 07 58

Screenshot 2023-05-17 at 17 08 10

**Curfew address check form accessed from the COM task list updated with new question:**
Screenshot 2023-05-17 at 17 07 30

**CA review page also updated with new question and warning:**
Screenshot 2023-05-17 at 17 04 05

### CODE REVIEW

File: server/routes/viewModels/taskLists/tasks/riskManagement.js [hunk 1]
```diff
@@ -29,8 +42,8 @@ const getLabel = ({ decisions, tasks }) => {
       return 'WARNING||Mandatory address checks not completed'
     }
 
-    if (awaitingRiskInformation) {
-      return 'WARNING||Still waiting for information'
+    if (warningLabel) {
+      return `WARNING||${warningLabel}`
```
Comment (louisenorris) [inline server/routes/viewModels/taskLists/tasks/riskManagement.js:46]: allows showing multiple warnings which is behaviour required if user selects `Yes` still awaiting other information and `No` to has the COM consulted the POM.

File: server/services/licence/licenceStatus.ts [hunk 2]
```diff
@@ -255,20 +257,24 @@ function getRiskManagementState(licence) {
   const checksConsideredAnswer = riskManagement?.hasConsideredChecks
   const awaitingInformationAnswer = riskManagement?.awaitingInformation || riskManagement?.awaitingOtherInformation
   const manageInTheCommunityAnswer = riskManagement?.manageInTheCommunity
-  const { proposedAddressSuitable, manageInTheCommunity } = riskManagement || {}
+  const pomConsultationAnswer = riskManagement?.pomConsultation
```
Comment (louisenorris) [inline server/services/licence/licenceStatus.ts:260]: boolean for use when determining if we should show the pom consultation warning.

File: server/services/licence/licenceStatus.ts [hunk 3]
```diff
@@ -255,20 +257,24 @@ function getRiskManagementState(licence) {
...
+      (riskManagementVersion === '2' || riskManagementVersion === '3') &&
+      checksConsideredAnswer !== 'Yes' &&
+      !bassRequested,
     proposedAddressSuitable:
       proposedAddressSuitable === 'Yes' || (riskManagementVersion === '3' && manageInTheCommunity === 'Yes'),
     awaitingRiskInformation: awaitingInformationAnswer === 'Yes',
     riskManagement: getState(),
     addressUnsuitable:
       proposedAddressSuitable === 'No' || (riskManagementVersion === '3' && manageInTheCommunity === 'No'),
+    pomNotConsulted: pomConsultation === 'No',
```
Comment (louisenorris) [inline server/services/licence/licenceStatus.ts:277]: boolean for use when determining if we should show the pom consultation warning.

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**